src/java.base/share/classes/java/security/KeyStore.java

```diff
@@ -145,21 +145,24 @@
  * <pre>
  *    KeyStore.ProtectionParameter protParam =
  *        new KeyStore.PasswordProtection(password);
+ *    try {
+ *        // get my private key
+ *        KeyStore.PrivateKeyEntry pkEntry = (KeyStore.PrivateKeyEntry)
+ *            ks.getEntry("privateKeyAlias", protParam);
+ *        PrivateKey myPrivateKey = pkEntry.getPrivateKey();
  *
- *    // get my private key
- *    KeyStore.PrivateKeyEntry pkEntry = (KeyStore.PrivateKeyEntry)
- *        ks.getEntry("privateKeyAlias", protParam);
- *    PrivateKey myPrivateKey = pkEntry.getPrivateKey();
+ *        // save my secret key
+ *        javax.crypto.SecretKey mySecretKey;
+ *        KeyStore.SecretKeyEntry skEntry =
+ *            new KeyStore.SecretKeyEntry(mySecretKey);
+ *        ks.setEntry("secretKeyAlias", skEntry, protParam);
  *
- *    // save my secret key
- *    javax.crypto.SecretKey mySecretKey;
- *    KeyStore.SecretKeyEntry skEntry =
- *        new KeyStore.SecretKeyEntry(mySecretKey);
- *    ks.setEntry("secretKeyAlias", skEntry, protParam);
- *
- *    // store away the keystore
- *    try (FileOutputStream fos = new FileOutputStream("newKeyStoreName")) {
- *        ks.store(fos, password);
+ *        // store away the keystore
+ *        try (FileOutputStream fos = new FileOutputStream("newKeyStoreName")) {
+ *            ks.store(fos, password);
+ *        }
+ *    } finally {
+ *        protParam.destroy();
```

Comment:
`KeyStore.ProtectionParameter` does not have a `destroy` method. Only `PasswordProtection` does.

Reply:
Oops, I tried to check but finally forgot about it.  Thanks!

Reply:
It's probably better to convert these long example code to snippets, maybe next time.

Reply:
The length is a little bit long, but it is fine to me.  It may be nice to revise the example and description, but let's do it later.  I modified the example by using one try clause only.